Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -457,10 +457,8 @@
         (extension-class "com.apple.app-sandbox.read-write" "com.apple.app-sandbox.read")
         (extension "com.apple.fileprovider.read-write")))
 
-(allow mach-lookup
-    (global-name "com.apple.logd")
-    (global-name "com.apple.logd.events")
-)
+(with-filter (require-not (state-flag "DisableLogging"))
+    (allow mach-lookup (global-name "com.apple.logd" "com.apple.logd.events")))
```

Comment:
Nit: Showing each service on its own line is more readable/easier to review.

Reply:
Fixed!